src/main/g8/default.properties

```diff
@@ -0,0 +1,5 @@
+name =
```

Comment:
Could this be renamed to something like "repo name", "project name" etc just so silly people like me (...hypothetically speaking of course 👀 ) don't think it's their own name that they should put?